3/collection-meta.yaml

```diff
@@ -14,3 +14,7 @@ collections:
     maintainers:
       - rndmh3ro
       - schurzi
+  kubernetes.core:
+    changelog-url: https://github.com/ansible-collections/community.kubernetes/blob/main/CHANGELOG.rst
+    maintainers:
+      - Ansible Cloud Team
```

Comment:
This needs to be github ids.  The idea is that we need at least one point of contact that we can get a hold of in case there's something wrong with the collection.